core-services/prow/02_config/openshift/lws-operator/_prowconfig.yaml

```diff
@@ -0,0 +1,14 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
```

Comment:
do we plan to make the requirements stricter later (e.g. jira/valid-bug)?

Reply:
I think it should already, but let me confirm

Reply:
Since we don't have older branches yet currently, there is no need to configure for `jira/valid-bug`. But I've updated the PR for `jira/valid-reference` in main branch.

Reply:
I see, thx!